spring-aop/src/main/java/org/springframework/aop/aspectj/VariableNameUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.aop.aspectj;
+
+import org.springframework.util.StringUtils;
+
+/**
+ * Utility class containing the logic to check if a string could be a Java identifier.
+ * variable name.
+ *
+ * @author Marten Deinum
+ * @since 5.3.2
+ * @see Character#isJavaIdentifierStart(char)
+ * @see Character#isJavaIdentifierPart(char)
+ */
+public abstract class VariableNameUtil {
```

Comment:
```suggestion
abstract class VariableNameUtil {
```

There's no need to make this utility class `public` -- right?

Reply:
Not really no. I initially figured to make it package protected/default scoped. But as most utility classes are `public` I figured lets make it public.